block-producer/src/store/mod.rs

```diff
@@ -93,18 +96,24 @@ impl ApplyBlock for DefaultStore {
 
 #[async_trait]
 impl Store for DefaultStore {
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(target = "miden-block-producer", skip(self, proven_tx), err)]
     async fn get_tx_inputs(
         &self,
         proven_tx: SharedProvenTx,
     ) -> Result<TxInputs, TxInputsError> {
-        let request = tonic::Request::new(GetTransactionInputsRequest {
+        let message = GetTransactionInputsRequest {
             account_id: Some(proven_tx.account_id().into()),
             nullifiers: proven_tx
                 .input_notes()
                 .iter()
                 .map(|nullifier| (*nullifier).into())
                 .collect(),
-        });
+        };
+
+        info!(target: target!(), ?message);
```

Comment:
Would this log `message` as a message or as a filed? If as a field, we should switch to message.